(AzureForums) Add a 'Note" to add app setting details

AjayKumar-N opened the pull request:
Ref doc: https://docs.microsoft.com/en-us/azure/nodejs-specify-node-version-azure-apps
The suggestions outlined in the document referenced, azuredeploy.json disregards WEBSITE_NODE_DEFAULT_VERSION variable during the deployment, a note could be added to inform the users to add an app setting as part of the site resource itself, as mentioned by David Ebbo in this sample: https://github.com/davidebbo/AzureWebsitesSamples/blob/331e48636a0d5b2d9b22c68961196b7040d0ab81/ARMTemplates/FunctionsWebDeploy.json#L44-L67

This request stems from the following WebApps forum thread, where this issue was discussed: https://social.msdn.microsoft.com/Forums/en-US/c2890f30-0ffa-4ec6-9f01-f5439389ac5a/inconsistent-nodejs-version-used-in-azure-documentation-bug?forum=windowsazurewebsitespreview

azurecla commented:
Hi __@szarkos__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (szark). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

AjayKumar-N committed:
(AzureForums) Add a 'Note" to add app setting details

Ref: https://docs.microsoft.com/en-us/azure/nodejs-specify-node-version-azure-apps
The suggestions outlined in the document referenced, azuredeploy.json disregards WEBSITE_NODE_DEFAULT_VERSION variable during the deployment, a note could be added to inform the customers/users to add an app setting as part of the site resource itself, as mentioned by David Ebbo in this sample: https://github.com/davidebbo/AzureWebsitesSamples/blob/331e48636a0d5b2d9b22c68961196b7040d0ab81/ARMTemplates/FunctionsWebDeploy.json#L44-L67
There has been a discussion on this query on the forum thread: https://social.msdn.microsoft.com/Forums/en-US/c2890f30-0ffa-4ec6-9f01-f5439389ac5a/inconsistent-nodejs-version-used-in-azure-documentation-bug?forum=windowsazurewebsitespreview
